docs/community/meetings/2024-04-11.md

```diff
@@ -0,0 +1,184 @@
+# 11 April 2024
+
+In this meeting we asked participants to suggest specific tools, templates, packages, etc, that we should include in our [Orientation guide](../../orientation/README.md).
+We used the high-level topics proposed in our [previous meeting](2024-02-19.md) as a starting point.
+
+Attendance: 7 in person, 9 online.
+
+!!! info "Git is my lab book updates"
+
+    We have switched to [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), which greatly improves the user experience.
+
+    For example, use the search bar above (press <kbd>F</kbd>) to interactively search the entire website.
+
+## Purpose of the guide
+
+We are aiming to keep the orientation guide short and simple, to avoid overwhelming newcomers.
+
+> James Ong: If we can agree on a structure, we can then get people to contribute to specific sections.
+
+> T.K. Le: schedule a one-hour meeting where everyone works on writing content for 30 minutes, and then reviews each others' content for 30 minutes?
```

Comment:
omg just noticed this "T.K." 😆

Reply:
🤦

Reply:
Fixed, and I can confirm that there are no more instances of "T.K." in the entire repository.